apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -55,10 +62,13 @@ interface Props {
 }
 
 const PollWorkerScreen: React.FC<Props> = ({
-  activateCardlessBallotStyleId,
+  activateCardlessBallot,
   resetCardlessBallot,
-  appPrecinctId,
+  appPrecinct,
   ballotsPrintedCount,
+  precinctId = appPrecinct.kind === PrecinctSelectionKind.SinglePrecinct
```

Comment:
it seems like this is just being defined based on whats gets passed in as appPrecinct so why are you explicitly passing it in in AppRoot?

Reply:
same confusion for me here.

Reply:
Reading the code I seem to remember that there's two different "precinct selection" things that could be happening. One is the admin configuration for what precincts the BMD can work with: all, or a single one. The second selection is for the poll worker screen and represents the currently selected precinct for activating a ballot when in "all precincts" mode. This is also setting it to be the selected precinct when in single precinct mode. There's probably a more elegant way to do this, but that's my understanding based on the hazy memory of writing it.